pkg/proxy/accesslog/log.go

```diff
@@ -20,6 +20,7 @@ import (
 	"os"
 
 	"github.com/cilium/cilium/pkg/lock"
+	"github.com/cilium/cilium/pkg/logfields"
```

Comment:
I would prefer to reduce dependencies from pkg/proxy/accesslog, i.e. cut the circular dependency the other way around. That package is a public API, and there are external dependencies on it, so it's highly desirable to limit the number of transitive dependencies.

Reply:
We could move the API definitions into a separate package if that helps, e.g. `pkg/proxy/accesslog/api`

Reply:
That would be great.
However, I looked into `pkg/logfields` and it's not too bad in terms of dependencies. It doesn't add additional transitive dependencies.

Reply:
Ironically, I had pointed logfields to accesslog to ensure we never accidentally broke the public API and because accesslog would be managed only within itself (and the logging convention I was trying to enforce was not as critical). With this change we potentially need a logfields owner to approve changes in accesslog but it is very unlikely.
If we want to further eliminate dependencies we can just set accesslog.FilePath and logfields.Path to the same value, and leave a comment to keep them in sync. shrug.

Reply:
Good point. Let's leave it as-is and revisit if that becomes an issue.